pyproject.toml

```
@@ -37,6 +37,7 @@ jsonschema2md = 'jsonschema2md:main'
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 PyYAML = "6.0.2"
+markdown = "^3.7"
```

```suggestion
markdown = "3.7"
```
Needed to make Renovate working well.